README.md

```diff
@@ -43,14 +41,16 @@ $ npm install dd-trace
 $ yarn add dd-trace
 ```
 
-However, existing projects that already use the `v4` release line, or projects that need to support EOL versions of Node.js, may continue to use these release lines.
+Existing projects that need to use EOL versions of Node.js may continue to use these older release lines.
 This is done by specifying the version when installing the package.
 
 ```sh
-$ npm install dd-trace@4
-$ yarn add dd-trace@4
+$ npm install dd-trace@4 # or whatever version you need
+$ yarn add dd-trace@4 # or whatever version you need
 ```
 
+Note, however, that the end-of-life release lines are no longer maintained and will not receive updates.
```

Comment:
We use a mix of `end-of-life`, `EOL` and `End of Life` in the README. Maybe we should be consistent?

Reply:
Fixed. It's now either `end-of-life` (with capitalization where needed for consistency), or `EOL` where abbreviations are appropriate, with the abbreviation defined.